### PR TITLE
Updates to pysat testing instruments

### LINKED ISSUE #86: Improve Instrument Filename Parsing in Testing Routines
Some older instrument routines still include a '/' in the date portion of the filename. This can produce problems on other systems when the associated load routine parses the filename. Update to using a '-' 
 instead of the '/' and be sure to remove the leading path when grabbing date information.

### PR DESCRIPTION
**Pull request by jklenzing**
Closes #86.

Summary of changes
- Updated pysat test instruments filenames to consistently use '%Y-%m-%d' format rather than '/' 
- Added docstring for load routines for all test instruments
- pep8 scrub

### CODE REVIEW

File: pysat/instruments/pysat_testing2d_xarray.py [hunk 1]
```diff
@@ -23,6 +23,26 @@ def init(self):
...
+        List of filenames
+    tag : (str or NoneType)
+        Instrument tag (accepts '' or a number (i.e., '10'), which specifies
+        the number of times to include in the test instrument)
+    sat_id : (str or NoneType)
+        Instrument satellite ID (accepts '')
+
+    Returns
+    -------
+    data : (pds.DataFrame)
```
Comment (rstoneback) [inline pysat/instruments/pysat_testing2d_xarray.py]: In this case it is an xarray

File: pysat/instruments/pysat_testing_xarray.py [hunk 2]
```diff
@@ -29,6 +29,32 @@ def init(self):
...
+    sim_multi_file_right : (boolean)
+        Adjusts date range to be 12 hours in the future or twelve hours beyond
+        root_date (default=False)
+    sim_multi_file_left : (boolean)
+        Adjusts date range to be 12 hours in the past or twelve hours before
+        root_date (default=False)
+
+    Returns
+    -------
+    data : (pds.DataFrame)
```
Comment (rstoneback) [inline pysat/instruments/pysat_testing_xarray.py]: xarray

File: pysat/instruments/pysat_testing2d_xarray.py (hunk 1 above)
Comment (jklenzing) [reply pysat/instruments/pysat_testing2d_xarray.py]: fixed

File: pysat/instruments/pysat_testing_xarray.py (hunk 2 above)
Comment (jklenzing) [reply pysat/instruments/pysat_testing_xarray.py]: fixed

**Pull request merged**